### ISSUE TITLE
Add timestamps to log output and configuration

### ISSUE THREAD

**Issue by mankinskin** [storyscript/tracing-wasm#10]
I would like the debug output to show timestamps just like the tracing logger on my server.
Example output from the server:
```
Sep 29 02:03:01.290 DEBUG server: Tracing initialized.
Sep 29 02:03:01.302  INFO server::server: Starting Server
Sep 29 02:03:07.758 DEBUG server::server: Websocket connection request
Sep 29 02:03:07.763 DEBUG server::server::websocket::connections: Opening Websocket connection 0
```
Current output by the wasm module (in the browser console):
```
DEBUG src/client/mod.rs:33
Tracing initialized. package.js:406
DEBUG src/client/mod.rs:45
Host: localhost:8000 package.js:406
DEBUG src/client/mod.rs:73
Root update package.js:406
DEBUG src/client/websocket.rs:91
Root update package.js:406
DEBUG src/client/websocket.rs:94
WebSocket opened package.js:406
DEBUG src/client/chart.rs:60
Subscriber received message
```
Something like this would be desirable:
```
Sep 29 02:03:06.612 DEBUG src/client/mod.rs:33
Tracing initialized. package.js:406
...
```

**Comment by mankinskin** [comment]
It would probably be best to rely on the tracing crate for this. I did some research and looked into how the `tracing_subscriber::fmt::Subscriber` in my server prints the timestamp. And I have found [`SubscriberBuilder::with_timer`](https://github.com/tokio-rs/tracing/blob/386969ba9ab6862f686a6aae60aa6b8781b91408/tracing-subscriber/src/fmt/mod.rs#L465) which is implemented when the `inner` [`fmt::Layer`](https://github.com/tokio-rs/tracing/blob/386969ba9ab6862f686a6aae60aa6b8781b91408/tracing-subscriber/src/fmt/fmt_layer.rs#L61) contains a [`Format`](https://github.com/tokio-rs/tracing/blob/386969ba9ab6862f686a6aae60aa6b8781b91408/tracing-subscriber/src/fmt/format/mod.rs#L177) event formatter as its `fmt_event` field. Then `Format` stores a Timer and uses it to write the time in its [`FormatEvent::format_event`](https://github.com/tokio-rs/tracing/blob/386969ba9ab6862f686a6aae60aa6b8781b91408/tracing-subscriber/src/fmt/format/mod.rs#L387) implementation.

So basically we should use a `tracing_subscriber::fmt::Layer` and initialize it `with_timer`? There is [`tracing_subscriber::fmt::time::time()`](https://github.com/tokio-rs/tracing/blob/386969ba9ab6862f686a6aae60aa6b8781b91408/tracing-subscriber/src/fmt/time/mod.rs#L42) to get a `SystemTime` timer. However that one might not work on wasm32. In that case we could use the [`tracing_subscriber::fmt::time::FormatTime`](https://github.com/tokio-rs/tracing/blob/386969ba9ab6862f686a6aae60aa6b8781b91408/tracing-subscriber/src/fmt/time/mod.rs#L22) trait to implement a timer that works on wasm32.

**Comment by mankinskin** [comment]
Since this crate does not already build on some common tracing Subscriber, it was simpler to just get the local time using chrono and formatting it into the log output like everything else. It might not be the most ambitious solution but the simplest.
